lib/src/components/NxRadio/NxRadio.tsx

```diff
@@ -30,8 +30,8 @@ const NxRadio = forwardRef<HTMLLabelElement, Props>(
                  onChange={() => onChange && onChange(value)}
                  readOnly={!onChange}/>
           <svg className="nx-radio__circle" viewBox="-8 -8 16 16" focusable={false}>
-            <circle r="7" strokeWidth="1" fill="none" className="nx-radio__outer-circle"/>
-            { isChecked && <circle r="5" stroke="none" className="nx-radio__inner-circle"/> }
+            <circle r="7" strokeWidth="1" className="nx-radio__outer-circle"/>
+            { isChecked && <circle r="5" strokeWidth="3" className="nx-radio__inner-circle"/> }
```

Comment:
With this new design, constructing the radio of two circles leads to a small visual issue, mostly noticeable when zoomed in as seem below.  It would probably be cleaner to just use one circle for the new design

Reply:
> With this new design, constructing the radio of two circles leads to a small visual issue, mostly noticeable when zoomed in as seem below. It would probably be cleaner to just use one circle for the new design
> ![image](https://user-images.githubusercontent.com/3630091/86596118-b9ebda80-bf67-11ea-8829-86154c2c0709.png)

This was really simple to fix with SCSS: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/e58b3889d67bac7d1aa35ac2eb1e670c33143afa

Reply:
> This was really simple to fix with SCSS: e58b388

That was a nicely minimal change from what was already there but it still results in SVG that's needlessly complicated for the current appearance.  How about this: https://github.com/sonatype/sonatype-react-shared-components/compare/RSC-120-NxRadio-redesign...RSC-120-NxRadio-redesign_ross

Reply:
> That was a nicely minimal change from what was already there but it still results in SVG that's needlessly complicated for the current appearance. How about this: RSC-120-NxRadio-redesign...RSC-120-NxRadio-redesign_ross

Oh a related question there: I noticed you set the interior to explicitly be white whereas it's just see-through in my impl.  Which should it be?

Reply:
> > This was really simple to fix with SCSS: [e58b388](https://github.com/sonatype/sonatype-react-shared-components/commit/e58b3889d67bac7d1aa35ac2eb1e670c33143afa)
> 
> That was a nicely minimal change from what was already there but it still results in SVG that's needlessly complicated for the current appearance. How about this: [RSC-120-NxRadio-redesign...RSC-120-NxRadio-redesign_ross](https://github.com/sonatype/sonatype-react-shared-components/compare/RSC-120-NxRadio-redesign...RSC-120-NxRadio-redesign_ross)

Merged with thanks.

Reply:
> > That was a nicely minimal change from what was already there but it still results in SVG that's needlessly complicated for the current appearance. How about this: RSC-120-NxRadio-redesign...RSC-120-NxRadio-redesign_ross
> 
> Oh a related question there: I noticed you set the interior to explicitly be white whereas it's just see-through in my impl. Which should it be?

I needed to make it white for my code to work, but there's no reason it cannot be transparent given your implementation